--- pom.xml	2021-10-21 17:15:50.000000000 +0900
+++ pom.xml.patched	2021-10-21 17:16:26.000000000 +0900
@@ -10,7 +10,7 @@
   <parent>
     <artifactId>master</artifactId>
     <groupId>org.sakaiproject</groupId>
-    <version>12-SNAPSHOT</version>
+    <version>12.6</version>
     <relativePath>../master/pom.xml</relativePath>
   </parent>
 
@@ -29,7 +29,7 @@
       <dependency>
         <groupId>org.sakaiproject</groupId>
         <artifactId>tincanapi-provider-impl</artifactId>
-        <version>12-SNAPSHOT</version>
+        <version>12.6</version>
       </dependency>
     </dependencies>
   </dependencyManagement>
