--- pom.xml	2021-10-21 17:09:06.000000000 +0900
+++ pom.xml.patched	2021-10-21 17:17:28.000000000 +0900
@@ -5,7 +5,7 @@
   <parent>
     <artifactId>tincanapi-provider-base</artifactId>
     <groupId>org.sakaiproject</groupId>
-    <version>12-SNAPSHOT</version>
+    <version>12.6</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   <name>Sakai tincanapi LRS Provider Impl</name>
